src/app/_components/Badge.tsx

```diff
@@ -1,12 +1,24 @@
-type BadgeProps = {
-  featured: boolean
-  children?: string
+interface BadgeProps {
+  children: React.ReactNode
+  badgeType: 'primary' | 'ghost'
 }
 
-export function Badge({ featured, children }: BadgeProps) {
-  return (
-    <span className="inline-block px-2 py-1 bg-white text-brand-800 uppercase rounded-xl text-sm font-semibold">
-      {featured ? 'Featured Project' : children}
-    </span>
-  )
+export function Badge({ children, badgeType }: BadgeProps) {
+  switch (badgeType) {
+    case 'primary': {
+      return (
+        <span className="inline-block max-w-fit rounded-lg bg-brand-300 px-2 py-1 text-center text-xs font-semibold text-brand-800">
+          {children}
+        </span>
+      )
+    }
+
+    case 'ghost': {
+      return (
+        <span className="inline-block max-w-fit rounded-lg border border-current bg-transparent px-2 py-1 text-center text-xs font-semibold text-brand-100">
+          {children}
+        </span>
+      )
+    }
+  }
 }
```

Comment:
We can refactor this - Something like:

```
const variantStyles = {
  primary:
    'bg-brand-300, text-brand-800',
  ghost:
    'bg-brand-800, text-brand-100',
}
```

Then use `clsx` to merge `baseStyles` and `variantStyles`.

~Please have a look at the `Button` component for a similar implementation.